internal/handlers/pg/pgdb/collections.go

```diff
@@ -32,7 +32,8 @@ import (
 )
 
 // validateCollectionNameRe validates collection names.
-var validateCollectionNameRe = regexp.MustCompile("^[a-zA-Z_-][a-zA-Z0-9_-]{0,119}$")
+// Empty collection name, names with `$` and `\x00` are not allowed.
+var validateCollectionNameRe = regexp.MustCompile("^[^$\x00]{1,235}$")
```

Comment:
After we validate collect name with regex, let's also check it with `utf8.ValidString`. Otherwise, we could have a lot of "fun" Unicode-related problems

Reply:
Added this